apps/zotonic_launcher/src/command/zotonic_cmd_addsite.erl

```diff
@@ -189,6 +192,14 @@ parse_args([ "-n", Schema | Args ], Acc) ->
     parse_args(Args, Acc#{ dbschema => Schema });
 parse_args([ "-a", Pw | Args ], Acc) ->
     parse_args(Args, Acc#{ admin_password => Pw });
+parse_args([ "-A", App | Args ], Acc) ->
+    case re:run(App, "^(true|false|1|0)$", [caseless, {capture, none}]) of
```

Comment:
I suppose this function can be simplified like(without using `re:run/3`):
```erlang
parse_args([ "-A", App | Args ], Acc) ->
    parse_args(Args, Acc#{ app => z_convert:to_bool(string:to_lower(App)) });
```

Reply:
The regex also checks match for "tRuE", "FaLsE", or any caseless. z_convert:to_bool/1 does not make these checks. Anyway, I think the check is needed because if there is no match an error is displayed and the command exits.
Without the regex, the option "-A foo" it's a valid option and the command proceeds as #{app => false}.